.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: npm CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
```

Comment:
I dont think we want PR here.  Just on pushes to main for this repo.

Reply:
removed pull_request